src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,51 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int buy = 0;
+        int supply = 0;
+        try (BufferedReader br = new BufferedReader(new FileReader(fromFileName))) {
+            while (true) {
```

Comment:
The use of `while (true)` is generally discouraged as it can lead to infinite loops if not handled properly. Consider using a more explicit loop condition.